zerver/lib/message.py

```diff
@@ -870,7 +861,7 @@ def do_render_markdown(
         mention_data=mention_data,
         email_gateway=email_gateway,
     )
-    return rendered_content
+    return rendering_result
```

Comment:
It is possible that following this change, we should delete/inline this function, since we've now deleted everything it does :).  Definitely a good follow-up commit project, to avoid extra complexity in this main commit.